native/mod_manager/mod_manager.c

```diff
@@ -1406,6 +1435,7 @@ static char *process_config(request_rec *r, char **ptr, int *errtype)
             }
             clean = 0;
             ap_assert(worker->s->port != 0);
+            /* XXX: needed >?? offset logic OK here, we save the worker information  */
```

Comment:
I'm not sure I understand the comment. Could you please clarify what do you mean by it?

Reply:
I think we can remove that code...
https://github.com/modcluster/mod_proxy_cluster/blob/main/native/mod_proxy_cluster/mod_proxy_cluster.c#L858
but the logic in mod_proxy_cluster needs to be checked before....

Reply:
Ok then.

Btw check you chat please :)